src/skins/vector/css/matrix-react-sdk/views/rooms/_Autocomplete.scss

```diff
@@ -38,17 +38,29 @@
 .mx_Autocomplete_Completion_pill {
     border-radius: 17px;
     height: 34px;
+    padding: 0px 5px;
     display: flex;
     user-select: none;
     cursor: pointer;
     align-items: center;
     color: $primary-fg-color;
 }
 
-.mx_Autocomplete_Completion_pill * {
+.mx_Autocomplete_Completion_pill > * {
```

Comment:
Hmm, I slightly question whether being in an autocomplete pill implies things will want this CSS, or whether it would be better to put this on the children directly, but if this works then fine.